src/index.ts

```diff
@@ -1,5 +1,6 @@
 import type { IncomingMessage, ServerResponse } from 'http';
 import { verify as edVerify } from 'noble-ed25519';
+import { Context, Next } from 'koa';
```

Comment:
We should avoid importing from koa here, as it forces every person to install koa even if they are not using it (I think this should be a type import)

Reply:
Yup, my bad, meant to use a type import here, simply forgot to change it to one when auto importing the types using VS Code.